packages/by-name/qemu-static/package.nix

```
@@ -71,10 +104,13 @@ in
         "-Dlinux_aio_path=${libaio}/lib"
         "-Dlinux_fdt_path=${patchedDtc}/lib"
       ];
-    patches = previousAttrs.patches ++ [
-      ./0001-avoid-duplicate-definitions.patch
-      # Based on https://github.com/NixOS/nixpkgs/pull/300070/commits/96054ca98020df125bb91e5cf49bec107bea051b#diff-7246126ac058898e6da6aadc1e831bb26afe07fa145958e55c5e112dc2c578fd.
-      # We applied the same change done to libaio to libfdt as well.
-      ./0002-add-options-for-library-paths.patch
-    ];
+    patches =
+      previousAttrs.patches
+      ++ [
+        ./0001-avoid-duplicate-definitions.patch
+        # Based on https://github.com/NixOS/nixpkgs/pull/300070/commits/96054ca98020df125bb91e5cf49bec107bea051b#diff-7246126ac058898e6da6aadc1e831bb26afe07fa145958e55c5e112dc2c578fd.
+        # We applied the same change done to libaio to libfdt as well.
+        ./0002-add-options-for-library-paths.patch
+      ]
+      ++ lib.optional snpSupport [ snpPatches ];
```

```suggestion
      ++ lib.optional snpSupport snpPatches;
```
`optional :: bool -> a -> [a]`